docs/userguide/dl4.rst

```diff
@@ -218,6 +218,28 @@ even fit convergence might be an issue for a large number of datasets.
 To strike a balance, what might be a practical solution for analysis of many runs is to
 stack runs taken under similar conditions and then do a joint fit on the stacked datasets.
 
+Serialisation of datasets
+-------------------------
+
+`~gammapy.datasets.MapDataset` and `~gammapy.datasets.MapDatasetOnOff` are serialised according to `GADF Sky Maps <https://gamma-astro-data-formats.readthedocs.io/en/v0.2/skymaps/index.html>`__.
```

Comment:
Maybe: The various `Maps` contained `~gammapy.datasets.MapDataset` and `~gammapy.datasets.MapDatasetOnOff` are serialized according